core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -1,10 +1,14 @@
 package scalaz.zio
 
+import scalaz.zio.ExitResult.Cause.Interruption
 import scalaz.zio.internal.Env
 
 trait RTS {
   lazy val env =
-    Env.newDefaultEnv(cause => IO.sync(println(cause.toString)))
+    Env.newDefaultEnv {
+      case Interruption => IO.unit // do not log interruptions
```

Comment:
Should it be `cause if cause.interrupted =>` ?

Reply:
Right, I didn’t consider nested interruptions. I’ll change.